server/src/modules/Question/question.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Controller, Get, Param, Post, Body, Put, Delete } from '@nestjs/common';
+import { QuestionDto } from './dto/question.dto';
+import { QuestionService } from './question.service';
+
+@Controller('question')
+export class QuestionController {
+
+    constructor(private questionService: QuestionService) {
+
+    }
+
+    @Get()
+    async findAll(): Promise<QuestionDto[]> {
+        return await this.findAll();
```

Comment:
Should it be this.questionService.findAll()?

Reply:
yep, fixed